dts/bindings/sensor/ltr,329.yaml

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) 2023 Konrad Sikora
+# SPDX-License-Identifier: Apache-2.0
+
+description: Lite-on LTR329 Digital Ambient Light Sensor
+
+compatible: "ltr,329"
+
+include: [sensor-device.yaml, i2c-device.yaml]
+
+properties:
+  gain:
+    type: int
+    default: 0
```

Comment:
Need to justify in the description why this default value was chosen. Is it a register reset value?

Reply:
register reset value, comment added